packages/pagination/src/PaginationControls.js

```diff
@@ -12,14 +12,14 @@ const PaginationControls = ({ directionLinks, autoHide, pageRange, marginPages,
   const { pageCount, currentPage, setPage } = usePagination();
 
   const createItem = (pageNumber) => (
-    <PaginationItem key={pageNumber} active={currentPage === pageNumber} data-testid={`control-page-${pageNumber}`}>
-      <PaginationLink
-        style={{ zIndex: 'auto' }}
-        onClick={() => setPage(pageNumber)}
-        type="button"
-        aria-label={`Go to page ${pageNumber}`}
-        aria-current={currentPage === pageNumber}
-      >
+    <PaginationItem
+      key={pageNumber}
+      active={currentPage === pageNumber}
+      data-testid={`control-page-${pageNumber}`}
+      aria-label={`Go to page ${pageNumber}`}
```

Comment:
I think this issue was already fixed and the aria attributes should be on the `PaginationLink`. The screenshot in the ticket has the aria attributes on the `li` which is what `PaginationItem` is. Where `PaginationLink` is a `button`.

Reply:
According to the 505 compliance ticket description: I had to change the the aria-label and aria-current from button tag to li tag, which was given in the code reference in for ticket PF-2339 and the old code had none of these changes required by the ticket. Screen-Shot for ref:

![image](https://user-images.githubusercontent.com/40965826/141486108-46a5deed-423d-4757-9aa1-a9c6f4550721.png)

Reply:
Also this bug ticket was created around end of July.

Since no changes is required, cancelling this PR.